### ISSUE TITLE
Enable automatic TeX equation numbering (stem blocks)

### ISSUE THREAD

**Issue by jxxcarlson** [asciidoctor/asciidoctor#1110]
I suggest a change to `MathJax.Hub.Config` so as to enable automatic 
equaiton numbering.  If this done, one can also refer to equations by user-defined labels.

This can be accomplished by adding the line

```
 TeX: { equationNumbers: { autoNumber: "AMS" } }
```

to `MathJax.Hub.Config`  with a preceding comma.  Then `MathJax.Hub.Config`  reads as follows.

```

<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [["\\(", "\\)"]],
    displayMath: [["\\[", "\\]"]],
    ignoreClass: "nostem|nolatexmath"
  },
  asciimath2jax: {
    delimiters: [["\\$", "\\$"]],
    ignoreClass: "nostem|noasciimath"
  },
  TeX: { equationNumbers: { autoNumber: "AMS" } }
});
</script>
```

I have tested this, and it works.  The above line of code comes from http://docs.mathjax.org/en/latest/tex.html, section title [Automatic Equation Numbering](http://docs.mathjax.org/en/latest/tex.html#automatic-equation-numbering).
